### DISCUSSION TITLE
What form of proxy is this?

### DISCUSSION THREAD

**Discussion by jet10000** [abhinavsingh/proxy.py#1348] [General]
```
https://www.proxy-site.com/https/77726476706e69737468656265137421f2f84e9669347d48771dc7af9758/book/list/
```
What form of proxy is this? What is it name? 

Can proxy.py implement this form of proxy?

Thanks.

**Comment by abhinavsingh** [comment]
@jet10000 Yes you should be able to implement such forms using `proxy.py`.   You will need custom client and `proxy.py` will allow you to write custom plugins to handle such navigational URLs

**Reply by jet10000** [reply]
Thanks.

After searching, I found that it is called "webvpn".

**Reply by jet10000** [reply]
https://community.synology.com/enu/forum/17/post/109224

The WebVPN is for the special case where you don't want a full peer-to-peer networking connection between your computer and the lan behind your Synology. The WebVPN allows allows the Synology router to become a relay point to pass web browser sessions between your computer (anywhere on the public Internet), and private web servers inside your local network "behind" the Synology router.

**Reply by jet10000** [reply]
```
http://127.0.0.1:8899/https/www.google.com/search?q=webvpn
```

Can proxy.py implement such a plug-in? If so, where to start?
